File: src/api/data_struct/TaskInfo.ts
import BoardInfo from "./BoardInfo";
import CheckStateInfo from "./CheckStateInfo";
import EndInfo from "./EndInfo";
import LimitInfo from "./LimitInfo";
import StartInfo from "./StartInfo";
import Task from "./Task";
import TaskTitleInfo from "./TaskTitleInfo";

export default class TaskInfo {
    public task: Task = new Task()
    public task_title_info: TaskTitleInfo = new TaskTitleInfo()
    public check_state_info: CheckStateInfo = new CheckStateInfo()
    public limit_info: LimitInfo = new LimitInfo()
    public start_info: StartInfo = new StartInfo()
    public end_info: EndInfo = new EndInfo()
    public board_info: BoardInfo = new BoardInfo()
}

File: src/app/task_info.go
package mi

type TaskInfo struct {
	Task           *Task           `json:"task"`
	TaskTitleInfo  *TaskTitleInfo  `json:"task_title_info"`
	CheckStateInfo *CheckStateInfo `json:"check_state_info"`
	LimitInfo      *LimitInfo      `json:"limit_info"`
	StartInfo      *StartInfo      `json:"start_info"`
	EndInfo        *EndInfo        `json:"end_info"`
	BoardInfo      *BoardInfo      `json:"board_info"`
}
